packages/jest-cli/src/test_path_pattern_to_regexp.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+module.exports = (testPathPattern: string) => new RegExp(testPathPattern, 'i');
```

Comment:
since we serialize/deserialize patterns when we spawn workers, i chose to have a single function that can produce consistent result on both sides (probably should put it in a comment in this file)